test/token/ERC20/SnapshotToken.test.js

```diff
@@ -0,0 +1,38 @@
+const SnapshotToken = artifacts.require('SnapshotTokenMock');
+
+contract('SnapshotToken', function ([_, account1, account2]) {
+  beforeEach(async function () {
+    this.token = await SnapshotToken.new({ from: account1 });
+  });
+
+  it('can snapshot!', async function () {
+    assert.equal(await this.token.balanceOf(account1), 10000);
```

Comment:
could we update this to use `should` assertions?

Reply:
although that may be annoying if `should.eventually.be.bignumber.eq()` doesn't work correctly